mandatory/2-function-creation.js

```diff
@@ -15,37 +21,93 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) {
+  if (totalPrice>200){
+    return totalPrice-(totalPrice * 0.1)
+  }else {
+    return totalPrice-(totalPrice *0.05)
+  }
+}
 
 /*
   Complete the function to print to the console the odd numbers between 1 and limit (use a while loop):
   */
-function printOddNumbers(limit) {}
+
+  function printOddNumbers(limit) {
+    let numberOfLoops= 1
+  while (numberOfLoops <limit) {
+    console.log(numberOfLoops);
+    numberOfLoops = numberOfLoops + 2;
+}
+}
```

Comment:
Comment: 
You have labelled the values different than me but the function is the same. Well done!

Mine:
function printOddNumbers(limit) {
 let num=1;
  while(num<limit){
    console.log(num);
    num=num+2;
  }
}